addon/adapters/rest-adapter.js

```diff
@@ -389,13 +403,19 @@ export default Adapter.extend(BuildURLMixin, {
     @return {Promise} promise
   */
   findAll(store, type, sinceToken, snapshotRecordArray) {
-    var query, url;
+    const url = this.buildURL(type.modelName, null, null, 'findAll');
+    const adapterOptions = snapshotRecordArray.adapterOptions || {};
+    const { include } = adapterOptions;
+
+    let query;
```

Comment:
Do you mind refactoring some of the duplicated code in `findRecord` and `findAll` into a helper method?